### PR TITLE
Add plot_raw_rho in parameters doc

### PR DESCRIPTION
**Pull request by NeilZaim**
Noticed that we had this option to plot the raw charge density but it's not included in the parameters documentation, so I've added it.

### CODE REVIEW

File: Docs/source/usage/parameters.rst [hunk 1]
```diff
@@ -1764,6 +1764,11 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Whether to include the guard cells in the output of the raw fields.
     Only works with ``<diag_name>.format = plotfile``.
 
+    * ``<diag_name>.plot_raw_rho`` (`0` or `1`) optional (default `0`)
+    By default, the charge density written in the plot files is averaged on the cell centers.
+    When ``<diag_name>.plot_raw_rho = 1``, then the raw (i.e. non-averaged) charge density also saved in the output files.
```
Comment (lucafedeli88) [inline Docs/source/usage/parameters.rst]:
```suggestion
    When ``<diag_name>.plot_raw_rho = 1``, then the raw (i.e. non-averaged) charge density is also saved in the output files.
```

**Review by lucafedeli88 — CHANGES_REQUESTED** [review]

File: Docs/source/usage/parameters.rst (hunk 1 above)
Comment (lucafedeli88) [reply Docs/source/usage/parameters.rst]: Does this mean that both the averaged and the non-averaged fields are saved on disk?

Comment (NeilZaim) [reply Docs/source/usage/parameters.rst]: Yes, provided that `rho` is included in `fields_to_plot` (both averaged and non-averaged are written independently).
Btw, I have tested this option and it seems to work.

**Review by lucafedeli88 — APPROVED** [review]
Thanks for this PR!

**Pull request merged**